api/v1alpha1/gateway_types.go

```diff
@@ -115,13 +115,17 @@ type Listener struct {
 	//
 	// +optional
 	TLS *ListenerTLS `json:"tls,omitempty" protobuf:"bytes,5,opt,name=tls"`
-	// Extension for this Listener.  The resource may be "configmap" (use
-	// the empty string for the group) or an implementation-defined resource
-	// (for example, resource "mylistener" in group "networking.acme.io").
+	// ExtensionRef for this Listener.  The resource may be "configmaps"
+	// (omit or use the empty string for the group) or an
```

Comment:
@Miciah IMO this is easier to read when "(omit or use the empty string for the group)" is removed.

Reply:
Updated.